### ISSUE TITLE
[Feature Request] Same-line trailing comment annotation parsing

### ISSUE THREAD

**Issue by JoGoFo-** [krzysztofzablocki/Sourcery#965]
It would be really cool if Sourcery could identify comment annotations at the _end_ of the line and associate it with the preceding statement.

For example:
```
let number: Int // sourcery: first = 1
```
and
```
enum Foo {
    case bar /* sourcery: first = "value" */
}
```
This would be helpful for cleaner formatting of code.

In the Writing Templates guide it says:
> Using /* and */ for annotation comment you can put it on the same line with your code. 

My initial interpretation of this was as described in above `Foo` example, it wasn't particularly clear that the annotation had to come _before_ the type.

I don't know how complex this would be to implement, I love this tool and would be more than happy to have a try myself if someone could guide me on how to get started. Tried cloning the repo already - the circleci config references a 'Sourcery.xcworkspace' which doesn't seem to exist anywhere, and the Rakefile _runs_ but fails a bunch of tests.

Thanks!

**Comment by krzysztofzablocki** [comment]
hey you want to open package.swift in Xcode and use that for development now, thanks for reminding me to update circle config 😅

AnnotationParser is a separate object you could try tweaking to support that logic, I'd be fine with same line being annotation like that being processed with the associated statement 👍

**Comment by art-divin** [comment]
Closing this one since it was implemented by #970
